include/iio/iio.h

```diff
@@ -1328,6 +1328,10 @@ struct iio_data_format {
 
 	/** @brief Number of times length repeats (added in v0.8) */
 	unsigned int repeat;
+
+	/** @brief Contains a value to be added to the raw sample before
+	 *  applying the scale. */
+	double offset;
```

Comment:
Just a side note... `structs` in header files are always nice to get some ABI breakage down the road. I wonder if we shouldn't have some `reserved` bits in here? At least for when v1.0 is out...

Reply:
Yeah I was thinking that this is not a problem because old applications won't try to access the "offset" field. Apps built with Libiio > 1.0 may try to access new fields that are yet to be added, but we never really cared about forward compatibility.

With that said - I can add some padding, it's not that much more work.

Reply:
Yeah, if the new fields are always added in the end of the struct but we will always leave the door open for subtle things and like you said... it's not much work at all

Reply:
Done, as a separate commit. Thanks for the review btw.